### ISSUE TITLE
Remove "git add" commands from lint-staged

### ISSUE THREAD

**Issue by epicfaace** [rjsf-team/react-jsonschema-form#1828]
I get this warning when committing:

```
@rjsf/core: ⚠ Some of your tasks use `git add` command. Please remove it from the config since all modifications made by tasks will be automatically added to the git commit index.
```

**Comment by Aadi-27** [comment]
Okay, I will pick this up

**Comment by OlushesiToheeb** [comment]
I'll like to work on this @epicfaace

**Comment by galanggg** [comment]
Hi @epicfaace , is this issue still available?

**Comment by epicfaace** [comment]
Yes, it is!

**Comment by OlushesiToheeb** [comment]
@galanggg are you working on this already? had forgot i commented on this

**Comment by galanggg** [comment]
I happen to be working on it, do you still want to take this issue? @OlushesiToheeb

**Comment by OlushesiToheeb** [comment]
@galanggg Yes I'll love to and @epicfaace I saw the` git add`  in  `core ` , `material ui` and `semantic ui`  `package.json lintstage` should I just remove it in all, push to a new branch and make a pull request?

**Comment by epicfaace** [comment]
Yes, please do!

**Comment by OlushesiToheeb** [comment]
@epicfaace they're some are I'm facing here , the first image was when trying to do `npm install` just prettier config but I got that error then the second was when i was trying to commit my changes. kindly check out and let me know what I should have done or what I'm doing wrong.
![image](https://user-images.githubusercontent.com/46800177/110411770-f6fcf500-803f-11eb-8c1f-395da061cb23.png)
![image](https://user-images.githubusercontent.com/46800177/110412080-7985b480-8040-11eb-8a9e-77a0ebb4210c.png)

**Comment by epicfaace** [comment]
Also, I think your first screenshot cuts off the actual error message. Can you send me what you meant?

**Comment by takumawatanabe-0920** [comment]
@epicfaace 
If you are still experiencing this bug, may I try?

**Comment by epicfaace** [comment]
Sure!

**Comment by v8686561728** [comment]
Hi @takumawatanabe-0920  

This issue has been resolved @OlushesiToheeb in this [PR]( https://github.com/rjsf-team/react-jsonschema-form/pull/2268) on Mar 20, 2021. 

@epicfaace could you please close this issue
